spec/Section 3 -- Type System.md

```diff
@@ -343,9 +343,17 @@ client-specific primitive for time. Another example of a potentially useful
 custom scalar is `Url`, which serializes as a string, but is guaranteed by
 the server to be a valid URL.
 
+When defining an additional scalar, GraphQL systems should use the `@specified`
+directive to provide an RFC3986-compliant URI pointing to a human-readable
```

Comment:
I think throughout we can replace `RFC3986-compliant URI` with just `URL`

`RFC3986-compliant URI` could be understood to include an ISBN for an academic paper or something like that, and the goal here is that a person can load this URL in a browser and go read the spec. `URL` is well understood with a single definition, so there's no need to qualify it further with which IETF RFC you're referring to.